docs/user/intro.md

```diff
@@ -23,6 +23,27 @@ import OcLogoLilac from '/static/img/oc-logo-lilac.svg';
 Here you will find all the important information to successfully take the first steps in OpenCloud, to be able to solve
 technical problems flexibly yourself and to be able to answer frequently asked questions.
 
+## What is OpenCloud?
+
+OpenCloud is the file sharing and collaboration solution of the Heinlein Group.
+
+Through intelligent file management and a strong open source community, files become valuable resources, effectively
+structured and usable in the long term. With flexible data rooms and intelligent access rights, teams can access and
+work together on data anytime, anywhere without barriers, but with a lot of productivity.
+
+## How can I use OpenCloud?
+
+Instructions on how to install and set up your own OpenCloud can be found in the ["Admin" Documentation](../admin/welcome/index.md) under ["Getting-Started"](../admin/getting-started/)
+
+## Documentation Versions
```

Comment:
can users know if they are on rolling or prod? if not, the explanation of rolling/prod would make no sense for a user - might even confuse them.